bot/settings.py

```diff
@@ -18,6 +18,10 @@
 
 AOC_SESSION_COOKIE = environ.get("AOC_SESSION_COOKIE")
 
+FRIENDO_API_USER = environ.get("FRIENDO_API_USER")
+FRIENDO_API_PASS = environ.get("FRIENDO_API_PASS")
+FRIENDO_API_URL = environ.get("FRIENDO_API_URL", "http://dev.friendo.us/api/")
```

Comment:
This needs to be documented in the README somewhere, probably in the current env setup instructions.